druntime/src/core/demangle.d

```diff
@@ -2901,3 +2911,63 @@ private string toStringConsume (immutable ManglingFlagInfo[] infos, ref ushort b
     }
     return null;
 }
+
+const CXX_DEMANGLER _no_cxx_demangler = cast(CXX_DEMANGLER) -1;
+shared CXX_DEMANGLER __cxa_demangle;
+
+/**
+ * Returns:
+ *  a CXX_DEMANGLER if a C++ stdlib is loaded
+ */
+
+CXX_DEMANGLER getCXXDemangler() nothrow @trusted
```

Comment:
Shouldn't those 3 symbols be `private` ?

Reply:
made all but this one `private`